### DISCUSSION TITLE
i need galaxy brain asap.

### DISCUSSION THREAD

**Discussion by itsRealM12C** [drknzz/GitHub-Achievements#366] [Q&A] [answered]
Hi. I wanna get galaxy brain because i wanna collect many achievements as i can. Regards.

**Comment by itsRealM12C** [comment]
okay.

**Reply by yokofajarsantosa** [reply]
Galaxy brain!!

**Comment by jvcByte** [comment]
Galaxy brain!!

**Comment by byKosta** [comment]
I'm also

**Comment by codewithpapakwame** [comment, marked as answer]
#366 Galaxy brain takes a long time TO GET IT SO YOU HAVE TO PUT THINKING CAPS ON WHEN YOU SEE FIRST DISCUSSION

**Reply by itsRealM12C** [reply]
**_### ALRIGHT._**

**Reply by codewithpapakwame** [reply]
also another one is go in a repo updating readme click pull request thing and then make sure is merged also 2 times to get badge also if you want to know my profile go to [Link](https://github.com/codewithpapakwame)

**Comment by andrewjswan** [comment]
This shouldn't be hard...

**Comment by codewithpapakwame** [comment]
Also another badge that is easy to get is close an issue or pr within 5 min

**Comment by Abdulaziz-developer1** [comment]
![image](https://github.com/user-attachments/assets/7be04923-8f9b-4254-9cef-f43d894fef0f)
I think that that is how it is got, upgraded, so pls accept my answer, also if you want additional info i can help you find it

**Comment by Abdulaziz-developer1** [comment]
also you can get Pull shark, by creating and merging pull request.
For you, i am gonna explain the easiest way to get it, first of all, enter one of your repositories.
then make a little change and then click 'commit changes'
![image](https://github.com/user-attachments/assets/9d581d06-d62f-41d0-b611-61a78c5b6d44)
choose the one on the down, then click the green button
![image](https://github.com/user-attachments/assets/182b7aa5-ed72-4054-a185-f7841fb23890)
here choose the 'create pull request'
after all
![image](https://github.com/user-attachments/assets/2612d6a4-cb04-4c1d-8f77-d9e649a95ecd)
click the merge pull request and do this process again, it takes ~ 2mins and you will get the pull shark

**Comment by Abdulaziz-developer1** [comment]
And here is how to earn more achievements
![image](https://github.com/user-attachments/assets/60d1c151-d3fb-4d72-b278-0bfdd277ddaf)
![image](https://github.com/user-attachments/assets/539e6b0a-f4df-4d9e-b448-03e8407eb9f1)

if needed i can send the link to read it yourself

**Comment by Abdulaziz-developer1** [comment]
pls accept 2 of them

I also need the achievement
